README.md

```diff
@@ -34,18 +34,46 @@ A transaction is displayed.
 
 ## For development
 
-You must use node v14.
+### Prerequisites
+
+* Install node v14
+* If you are using `WSL` (Windows Subsystem for Linux):
```

Comment:
I feel we should move this `WSL` text to the `Troubleshooting` section. This is not common and it's better to be in a separate place.

Reply:
Ok, ok!